mvpa2/base/constraints.py

```diff
@@ -10,16 +10,20 @@
 
 # TODO: __str__ / __repr__'s
 
+from builtins import map
+from builtins import str
+from past.builtins import basestring
+from builtins import object
```

Comment:
ha ha -- I can answer my own question which came because I am (was) primarily six user, and not of the future.  So `builtins` (and the `past`) is the compatibility layer from the `future` package.  Thanks ;)